### ISSUE TITLE
Have you tried large-scale input?

### ISSUE THREAD

**Issue by shuoshuoa** [ZJULearning/ttfnet#19]
I tried to use 1333x800 and keep_ratio, the backbone is darknet53, other parameters have not changed, the result is not good, map = 30.6; have you tried? Why the effect is so bad?

**Comment by liuzili97** [comment]
No, our gpus can not fit 12 images of 1333x800. I only have tried 608x608 and 640x640 before. The AP is better but the speed is much slower.

**Comment by shuoshuoa** [comment]
Thanks for your quick reply, I'm guessing if it is not enough for the classification task to train only the central point for large-size images
